### ISSUE TITLE
Does this support MySQL?

### ISSUE THREAD

**Issue by rw3iss** [andrglo/mssql-cr-layer#1]
I know MS SQL and MySQL and almost identical. 
I'm wondering if the mssql-cr-layer can be used on a mysql database?
If not, do you think it would be hard to modify it to do so (I can try)?
Thanks.

**Comment by andrglo** [comment]
No, this API is specific to mssql. I created this package, and the pg-cr-layer because I needed a unified api to access sql server and postgres. For MySQL it would be required to create a new repository called mysql-cr-layer. I can create this new repository, copy this code there and make the initial adjustments. Then you continue.
